README.md

```diff
@@ -12,52 +12,171 @@
 [greenkeeper-image]:https://badges.greenkeeper.io/moxystudio/react-with-moxy.svg
 [greenkeeper-url]:https://greenkeeper.io
 
-MOXY's react boilerplate.
+Opinionated boilerplate to create isomorphic react applications.
 
 ## Motivation
+Isomorphic JavaScript applications can be pretty challenging and hard to setup.
+
+REACT WITH MOXY offers you the required tooling for your universal JavaScript application, aswell as an opinionated frontend stack ready to kick-off your next project.
+
+We aim to remove all the tooling to [react-scripts](https://github.com/moxystudio/react-with-moxy-scripts), which will abstract all the complexity behind setting up webpack, server-side rendering and other complex tasks.
+
+## What’s Included?
+- [React](https://reactjs.org/)
+- [React Router](https://github.com/ReactTraining/react-router)
```

Comment:
We should mention that we are using v3 because v4 have some conceptual problems that we are trying to avoid. See https://github.com/ReactTraining/react-router/issues/5076#issuecomment-326456921